docs/api-overview.md

```diff
@@ -24,9 +24,28 @@
 
 # API documentation
 
-The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM.
+The Eclipse OpenJ9 VM provides supplementary application programming interfaces and extensions, which can be used to improve performance, assist with problem determination, or help monitor and manage the OpenJ9 VM. The documentation also includes links to the API documentation for the Java&trade; SE and JDK reference implementation.
+
+
+## Performance

```

Comment:
Performance strikes me as the wrong title for the two items that follow.  It's too general given the specific areas - native data and gpu - mentioned.

Can we give each of the following two paragraphs their own title?  ie: `Native data interop` & `GPU acceleration`?

Reply:
Done